monitoring/monitorlib/scd_automated_testing/scd_injection_api.py

```diff
@@ -38,14 +38,15 @@ class FlightAuthorisationData(ImplicitDict):
 class InjectFlightRequest(ImplicitDict):
     ''' A class to hold the details of a test injection payload '''
     operational_intent: OperationalIntentTestInjection
-    flight_authorisation: FlightAuthorisationData
+    flight_authorisation: Optional[FlightAuthorisationData]
```

Comment:
I agree that the first thought should probably be that if the uss_qualifier isn't checking for U-space compliance, the flight authorisation information may not be necessary.  However, I can imagine a circumstance in which some USSs have designed their systems with U-space in mind, yet we don't want to check for U-space compliance.  In that case, it would still be very useful to the USSs to have U-space information provided to them since they're designed to use it.  And, it's no problem for uss_qualifier to generate this mock data, so I'd suggest we provide all information all the time from uss_qualifier.  Concretely, this means we would provide flight_authorisation even if the uss_qualifier wasn't going to validate anything related to that information, and even if that information wasn't relevant in the locality for which the test was being conducted (e.g., US).  In those cases, it's still easy for the USS to ignore the flight_authorisation field.

So, I'd recommend we keep this field as non-Optional.

Reply:
Thank you for the clarification, I reverted this change and updated the test_definitions.